packages/mui-material/src/Tab/Tab.d.ts

```diff
@@ -75,4 +75,6 @@ export type TabProps<
   P = {},
 > = OverrideProps<TabTypeMap<P, D>, D>;
 
+export interface TabsPropsIndicatorColorOverrides {}
```

Comment:
This is not enough. You have to combine it with `OverridableStringUnion`.

Here is an example of the button variant:

```ts
   variant?: OverridableStringUnion<
      'text' | 'outlined' | 'contained',
      ButtonPropsVariantOverrides
    >;
```

Reply:
Thank your for your review, sorry for missing this. It's my first pull request ever and I'm still trying to get familiar with this project 😅 
I will fix this later this day

Reply:
No worries! We are really appreciate your help.

Reply:
You mean like this?

`indicatorColor?: OverridableStringUnion<
      'primary' | 'secondary',
      TabsPropsIndicatorColorOverrides
    >;`

Reply:
@AHeiming Are you able to update what we suggested? let me know if you need any more help.

Reply:
@AHeiming Will you be working on this further?